duckduckgo/iOS
# update dax dialog asset

Task/Issue URL: https://app.asana.com/0/392891325557410/1200036270122429
Tech Design URL:
CC:

**Description**:

Update Dax Dialog asset to be consistent with the rest of the app and Android.

**Steps to test this PR**:
1. Clean install, run through onboarding and Dax Dialogs, make sure it looks OK.


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
